engine/src/test/java/io/zeebe/engine/util/client/JobClient.java

```diff
@@ -87,6 +87,11 @@ public JobClient withRetries(final int retries) {
     return this;
   }
 
+  public JobClient withBackOff(final long backOff) {
```

Comment:
The test could be easier to read if we pass the backoff as a `Duration` object.

```suggestion
  public JobClient withBackOff(final Duration backOff) {
```

Reply:
Done